frontend/src/components/molecules/MatchItem/MatchItem.tsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import styled from "styled-components";
+import ListItemAvatar from '@material-ui/core/ListItemAvatar';
+import ListItemText from '@material-ui/core/ListItemText';
+import {Avatar, Card, Typography, withStyles} from '@material-ui/core';
+import {MatchTeam} from "../MatchTeam/MatchTeam";
+import Accordion from '@material-ui/core/Accordion';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import {THEME} from "../../atoms/constants/ThemeMUI";
+import AddCircleTwoToneIcon from '@material-ui/icons/AddCircleTwoTone';
+import MatchWinnerDeterminationDialog from "../MatchWinnerDeterminationDialog/MatchWinnerDeterminationDialog";
+
+export type MatchItemProps = {
+    onClickTeam: () => void,
```

Comment:
onClickTeam? To nie chcemy miec w argumencie funkcji, jaka druzyna zostala kliknieta? Po co to jest?

Reply:
Chciałbym to poprawić dopiero po połączeniu z Backendem - nie do końca jeszcze jestem pewny. Wtedy chciałbym nanieść poprawki do tego. 
Możliwe, że jeszcze tego dobrze nie widzę.
Zapewne masz rację.